Documentation/Diagnostics/PH2094.md

```diff
@@ -0,0 +1,51 @@
+# PH2094: Prefer using the named tuple field, not ItemX
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2094 |
+| Category  | [Readability](../Readability.md) |
+| Analyzer | [PreferTupleFieldNamesAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Readability/PreferTupleFieldNamesAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+Wherever possible, use the provided name of a tuple field, not the generic name (`Item1`, `Item2`, ...). 
+
+## How to solve
+
+See [MS learn](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples#tuple-field-names)
+
+## Example
+
+Code that triggers a diagnostic:
+``` cs
+class BadExample
+{
+    public int BadMethod()
+    {
```

Comment:
Consider adding sum and count here (but still reference Item1)

Reply:
Agree, we should focus on the correct Diagnostic here